consumer/src/main/kotlin/consumer/FooClass.kt

```diff
@@ -4,6 +4,7 @@ import arrow.Kind
 import arrow.`*`
 
 /** HigherKinds **/
+/** AutoFold **/
```

Comment:
This class has been removed in master and each plugin has its own test class only with the declarations it needs. We need to remove all references to kinds and other plugins from this file that are not used in the autofold plugin